code/game/objects/structures/gym/weight_machine.dm

```diff
@@ -31,18 +31,19 @@
 		"You feel robust!",
 		"You feel indestructible!",
 	)
+	var/static/list/finished_silicon_message = list(
+		"You feel nothing!",
+		"No pain, no gain!",
+		"Your shell is hardened... not!",
```

Comment:
`Your shell is hardened... not!`

This line feels a bit weak. How about

`Chassis hardness rating...Unchanged.`

Reply:
Yeah sounds good, I'm terrible at writing flavortext stuff